src/components/containers/SingleArticle.jsx

```diff
@@ -107,6 +107,7 @@ class SingleArticle extends Component {
       || !this.props.singleArticle.article.articleBody
     ) return delayDisplay;
 
+
     return (
       <>
```

Comment:
What's this really? 🤔

Reply:
Fragment. To hold them all in one container instead of having too many nested divs.

Reply:
Got it.